File: VB/DxSample/FilterAlignmentConverter.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Data
Imports DevExpress.Xpf.Grid
Imports System.Windows.Controls

Namespace DxSample
	Friend Class FilterAlignmentConverter
		Implements IValueConverter
		Public Function Convert(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements IValueConverter.Convert
			Dim column As GridColumn = TryCast((TryCast(value, GridColumnHeader)).DataContext, GridColumn)
			Return column.GetValue(FilterAlingment.FilterAlingmentProperty)
		End Function

		Public Function ConvertBack(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements IValueConverter.ConvertBack
			Throw New NotImplementedException()
		End Function
	End Class
End Namespace

File: CS/DxSample/FilterAlignmentConverter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Data;
using DevExpress.Xpf.Grid;
using System.Windows.Controls;

namespace DxSample
{
    class FilterAlignmentConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
        {
            GridColumn column = (value as GridColumnHeader).DataContext as GridColumn;
            return column.GetValue(FilterAlingment.FilterAlingmentProperty);
        }

        public object ConvertBack(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
        {
            throw new NotImplementedException();
        }
    }
}
